src/components/policy/policy_regular/include/policy/sql_pt_representation.h

```diff
@@ -52,6 +52,7 @@ namespace policy {
 class SQLPTRepresentation : public virtual PTRepresentation {
  public:
   SQLPTRepresentation();
+  explicit SQLPTRepresentation(bool in_memory);
```

Comment:
@ZhdanovP propose to use enum against bool. Even if in extended policy is not like that

Reply:
@LuxoftAKutsan why do we need enum here? Except this, commits were updated.

Reply:
@ZhdanovP  for better understanding from caller code. 
Compare 
 - `SQLPTRepresentation db(true)` with `SQLPTRepresentation db(SQLPTRepresentation::IN_MEMORY)`
- Compare `SQLPTRepresentation db(false)` with `SQLPTRepresentation db(SQLPTRepresentation::IN_FILE_SYSTEM)`

But I propose to create task for it as additional enhancement